### PR TITLE
Update GSoC-interest.md

### PR DESCRIPTION
**Pull request by apoorvkhare07**
microtask 1 and 2 completed.

### CODE REVIEW

**Comment by GeorgLink** [comment]
Thanks for the changes @apoorvkhare07 
Can you please resolve the conflict? I accepted another pull request and you need to make sure to incorporate the latest change.

**Comment by apoorvkhare07** [comment]
Conflicts resolved @GeorgLink 
Please check if everything works fine now.

File: GSoC-interest.md [hunk 1]
```diff
@@ -22,5 +22,5 @@ GSoC Mentors
 | Pranjal Aswani | aswani.pranjal at gmail.com | Reporting of CHAOSS Metrics | [aswanipranjal/chaoss-microtasks](https://github.com/aswanipranjal/chaoss-microtasks) | To-Do |
 | Jacek Dąbrowski | j.dabrowski at cs.ucl.ac.uk | Improve Manuscripts and provide with new CHAOSS Metrics | [jsdabrowski/CHAOSS-project](https://github.com/jsdabrowski/CHAOSS-project) | To-Do |
 | Sai Varun Kolluru | saivarunkolluru at gmail.com | Reporting of CHAOSS Metrics | [f2014169/Chaoss-Micro-Tasks](https://github.com/f2014169/Chaoss-Micro-Tasks) | To-Do |
-| Apoorv Khare | apoorvkhare007 at gmail.com | Reporting of CHAOSS Metrics | [apoorvkhare07/Chaoss-Microtasks](https://github.com/apoorvkhare07/Chaoss-Microtasks)| To-Do |
-
+| Jaishree Dhage | jaishree.babu96 at gmail.com | Reporting of CHAOSS Metrics | [jaishreedhage/Chaoss](https://github.com/jaishreedhage/Chaoss) | To-Do |
```
Comment (GeorgLink) [inline GSoC-interest.md]: Can you please switch the order and add yourself to the end of the table?

**Comment by apoorvkhare07** [comment]
I have done the same in the last commit @GeorgLink 
https://github.com/chaoss/governance/pull/14/files#diff-34f696032839568d452502e3ab4924cfR26

**Comment by apoorvkhare07** [comment]
Thanks @GeorgLink .
Had a query regarding microtasks evaluation. 
Will you, along with other mentors, be reviewing the tasks and suggesting changes ?

**Comment by germonprez** [comment]
@apoorvkhare07 we will be reviewing your submissions. This will most likely include the suite of mentors who are involved in GSoC. I don't know that we will be suggesting changes to the tasks as the time window for selection is short.

**Pull request merged**